docs/src/pages/docs/functions/predicate-functions.md

```diff
@@ -31,6 +31,8 @@ description of their truth:
 * `isDefined :: a -> Boolean`: Every value that is not `undefined`, `null` included
 * `isEmpty :: a -> Boolean`: Empty Monoid, Object, Array or String
 * `isExtend :: a -> Boolean`: an ADT that provides `map` and `extend` methods
+* `isFalse :: a -> Boolean`: a value that is strictly equal to `false`
+* `isFalsy :: a -> Boolean`: a value that is considered to be [`falsy`](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```

Comment:
This needs to be [](). Just tested and confirmed. The []() is the standard way to do it, [][] is for when your referencing the thinks at the bottom of the `md`

Reply:
yep exaclty this would be something like:
```markdown
[`falsy`][falsy-link]

[falsy-link]: https://developer.mozilla.org/en-US/docs/Glossary/Falsy
```

That makes it easier to change and verify links

Reply:
gotcha, i get what you're asking for now, i misunderstood previously